### DISCUSSION TITLE
How to change the date/time for the system (or a single game)?

### DISCUSSION THREAD

**Discussion by jackmillward** [flyinghead/flycast#1154] [Q&A] [answered]
Hello all, I'm playing a game that kind of needs you to be able to progress time quickly, with the game using your system date/time.

I've changed the system time and date in the Dreamcast BIOS, but it doesn't seem to be working when I launch up the game, and it shows my system date rather than the one set in the BIOS. Changing the date/time in the BIOS doesn't save if you restart the BIOS.

I've also checked the configuration file at `<HOME>/.config/flycast/emu.cfg` and can't spot any obvious configuration in there.

Is there a way for this to be done without changing my actual system date/time?

**Comment by flyinghead** [comment, marked as answer]
The host system current date/time is used by Flycast to set the dreamcast clock when starting a game. There's currently no way to avoid this or set a custom date instead.
The only workaround I can think of is to change the host system time before starting the game. (You can revert it to normal once the game runs.)

**Reply by jackmillward** [reply]
Brilliant, thanks very much. Great project you have here, by the way.

The system time thing you mentioned does work, but can interfere with some other applications. Is there anything on a roadmap for a persistent bios or saved settings? Sort of like a Dreamcast console would be, with saved system settings?
